src/client/screens/UserAccountScreen.tsx

```diff
@@ -0,0 +1,319 @@
+    fontSize: 1
+  },
+  textBox: {
+    fontSize: 1,
+    fontWeight: "normal"
+  },
+  header: {
+    mb: "3"
+  },
+  userFields: {
+    flexDirection: "column",
+    width: "300px",
+    mr: 3
+  }
+};
+
+interface UserForm {
+  readonly name: string;
+  readonly email: string;
+  readonly isMarketingEmailOn: boolean;
+}
+
+interface ValidUserForm extends UserForm {
+  readonly valid: true;
+}
+
+interface InvalidUserForm extends UserForm {
+  readonly valid: false;
+}
+
+const accountDeleteEmail = "districtbuilder@azavea.com";
+
+const validate = (form: UserForm) =>
+  form.name.trim() !== "" && form.email.trim() !== ""
+    ? ({ ...form, valid: true } as ValidUserForm)
+    : ({ ...form, valid: false } as InvalidUserForm);
+
+const UserAccountScreen = () => {
+  const user = useSelector((state: State) => state.user);
+  const dispatch = useDispatch();
+  const [updateUserResource, setUpdateUserResource] = useState<WriteResource<UserForm, IUser>>({
+    data: {
+      name: "",
+      email: "",
+      isMarketingEmailOn: false
+    }
+  });
+  const { data } = updateUserResource;
+
+  useEffect(() => {
+    dispatch(userFetch());
+  }, []);
+
+  useEffect(() => {
+    if ("resource" in user) {
+      setUpdateUserResource({
+        data: {
+          ...data,
+          name: user.resource.name,
+          email: user.resource.email,
+          isMarketingEmailOn: user.resource.isMarketingEmailOn
+        }
+      });
+    }
+  }, [user]);
+
+  const isUpdatePending = useMemo(
+    () => "isPending" in updateUserResource && updateUserResource.isPending,
+    [updateUserResource]
+  );
+
+  const isChanged = useMemo(() => {
+    if ("resource" in user) {
+      return (
+        updateUserResource.data.name.trim() !== user.resource.name ||
+        updateUserResource.data.isMarketingEmailOn !== user.resource.isMarketingEmailOn
+      );
+    }
+    return false;
+  }, [updateUserResource.data, user]);
+
+  const isSubmitDisabled = useMemo(
+    () =>
+      !isChanged ||
+      "errorMessage" in updateUserResource ||
+      !validate(data).valid ||
+      isUpdatePending,
+    [isChanged, data, updateUserResource, isUpdatePending]
+  );
+
+  const onNameChange = (e: React.ChangeEvent<HTMLInputElement>) =>
+    setUpdateUserResource({
+      data: { ...data, name: e.currentTarget.value }
+    });
+
+  const onCommunicationChange = () =>
+    setUpdateUserResource({
+      data: {
+        ...data,
+        isMarketingEmailOn: !data.isMarketingEmailOn
+      }
+    });
+
+  const onSubmit = async (e: React.FormEvent) => {
+    e.preventDefault();
+    const validatedForm = validate(data);
+    if (validatedForm.valid === true) {
+      setUpdateUserResource({ data, isPending: true });
+      const { name, isMarketingEmailOn } = validatedForm;
+      try {
+        const user = await patchUser({ name, isMarketingEmailOn });
+        setUpdateUserResource({ data, resource: user });
+        dispatch(userFetch());
+      } catch (errors: any) {
```

Comment:
Since we don't log or capture the error, we don't need to catch it (which will also get rid of a warning about using the `any` type).
```suggestion
      } catch {
```